src/System.Buffers/ref/project.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+      "System.Runtime": "4.4.0-beta-24625-01"
+    },
+  "frameworks": {
+    "netstandard1.1": {}
```

Comment:
Given that this will be the reference assembly for Netstandard1.7, shouldn't we have `netstandard1.7` in here instead?